docker/README.md

```diff
@@ -5,46 +5,117 @@ Thats right folks onedrive is now dockerized ;)
+
+The second one is your data folder that needs to sync with. Keep in mind that:
+
+-   The owner of the folder must not be root
+
+-   The owner have permission to its parent directory  
+    (because onedrive will try to setup a monitor for the sync folder).
+
+### 3. First run
+
+Onedrive also needs to be authorized with your account.  
+This is done by running docker in interactive mode. 
+
+**make sure to change onedriveDir to your own.**
+
+```bash
+onedriveDir="${HOME}/OneDrive"
+docker run -it --restart unless-stopped --name onedrive -v onedrive_conf:/onedrive/conf -v "${onedriveDir}:/onedrive/data" driveone/onedrive
 ```
-## Poweruser section
-1.  Check if monitor service is running
+
+-   You will be asked to open a specific link using your web browser 
+-   login into your Microsoft Account and give the application the permission  
+-   After giving the permission, you will be redirected to a blank page.  
+-   Copy the URI of the blank page into the application.
+
+If your onedrive is working as expected, you can detach from the container with Ctrl+p, Ctrl+q.
+
+### 4. Status, stop, and restart
+
+Check if monitor service is running
+
 ```bash
 docker ps -f name=onedrive
 ```
-2.  Show monitor run logs
+
+Show monitor run logs
+
 ```bash
 docker logs onedrive
 ```
-3.  Stop running monitor
+
+Stop running monitor
+
 ```bash
 docker stop onedrive
 ```
-4.  Resume monitor
+
+Resume monitor
+
 ```bash
 docker start onedrive
 ```
-5.  Unregister onedrive monitor
+
+Unregister onedrive monitor
+
 ```bash
 docker rm -f onedrive
 ```
+
+### 5. Edit the config
+
+Onedrive should run in default configuration, but however you can change your configuration.  
+
+First download the default config from [here](https://raw.githubusercontent.com/abraunegg/onedrive/master/config)  
+Then put it into your onedrive_conf volume: `/var/lib/docker/volumes/onedrive_conf/_data`
+
+Or you can map your own config folder to config volume (copy stuffs from docker volume first)
+
+The detailed document for the config can be found here: [additional-configuration](https://github.com/abraunegg/onedrive#additional-configuration)
+
+## Run or update with one script
+
+If you are experienced with docker and onedrive, you can use the following script:
+
+```bash
+# Update onedriveDir with correct existing OneDrive directory path
+onedriveDir="${HOME}/OneDrive"
+
+firstRun='-d'
+docker pull driveone/onedrive
+docker inspect onedrive_conf > /dev/null || { docker volume create onedrive_conf; firstRun='-it'; }
+docker inspect onedrive > /dev/null && docker rm -f onedrive
+docker run $firstRun --restart unless-stopped --name onedrive -v onedrive_conf:/onedrive/conf -v "${onedriveDir}:/onedrive/data" driveone/onedrive
+```
+
 ## Build instructions
+
 ```bash
-cd docker
 git clone https://github.com/abraunegg/onedrive
-docker build . -t driveone/onedrive
+cd onedrive
+docker build . -f docker/Dockerfile -t driveone/onedrive
```

Comment:
This is still wrong, I don't know how you confirmed that:
```
$ git clone https://github.com/abraunegg/onedrive
Cloning into 'onedrive'...
remote: Enumerating objects: 35, done.
remote: Counting objects: 100% (35/35), done.
remote: Compressing objects: 100% (28/28), done.
remote: Total 2194 (delta 20), reused 13 (delta 7), pack-reused 2159
Receiving objects: 100% (2194/2194), 1.35 MiB | 1.19 MiB/s, done.
Resolving deltas: 100% (1446/1446), done.
$ cd onedrive/
$ docker build . -f docker/Dockerfile -t driveone/onedrive
Sending build context to Docker daemon  2.376MB
Step 1/9 : FROM centos
 ---> 1e1148e4cc2c
Step 2/9 : ENV GOSU_VERSION=1.11
 ---> Using cache
 ---> d396fb59b038
Step 3/9 : RUN yum install -y make git gcc libcurl-devel sqlite-devel &&     rm -rf /var/cache/yum/ &&     curl -fsS -o install.sh https://dlang.org/install.sh &&     bash install.sh dmd &&     gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4     && curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/${GOSU_VERSION}/gosu-amd64"     && curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/${GOSU_VERSION}/gosu-amd64.asc"     && gpg --verify /usr/local/bin/gosu.asc     && rm /usr/local/bin/gosu.asc     && rm -r /root/.gnupg/     && chmod +x /usr/local/bin/gosu     && gosu nobody true
 ---> Using cache
 ---> 36eaab54ecc1
Step 4/9 : RUN mkdir -p /onedrive/conf /onedrive/data
 ---> Using cache
 ---> 2bc7251c341f
Step 5/9 : ADD onedrive /usr/src/onedrive
ADD failed: stat /var/lib/docker/tmp/docker-builder561614743/onedrive: no such file or directory
$
```

Reply:
I'm really sorry I forgot I modified the dockerfile